src/multivariate/mvnormalcanon.jl

```diff
@@ -174,6 +174,7 @@ sqmahal!(r::AbstractVector, d::MvNormalCanon, x::AbstractMatrix) = quad!(r, d.J,
 unwhiten_winv!(J::AbstractPDMat, x::AbstractVecOrMat) = unwhiten!(inv(J), x)
 unwhiten_winv!(J::PDiagMat, x::AbstractVecOrMat) = whiten!(J, x)
 unwhiten_winv!(J::ScalMat, x::AbstractVecOrMat) = whiten!(J, x)
+unwhiten_winv!(J::PDSparseMat, x::AbstractVecOrMat) = x[:] = J.chol.L' \ x
```

Comment:
Isn't the lower triangular matrix _L_ from a Cholesky decomposition the transpose of the upper triangular matrix _U_? Seems like this could instead be written
```suggestion
unwhiten_winv!(J::PDSparseMat, x::AbstractVecOrMat) = x[:] = J.chol.U \ x
```
unless I'm missing something.

Reply:
You're not missing anything, this is correct!